ReactWindows/ReactNative.Net46.Tests/Internal/DispatcherHelpers.cs

```diff
@@ -14,7 +14,7 @@ static class DispatcherHelpers
             Dispatcher dispatcher = Application.Current != null
                 ? Application.Current.Dispatcher
                 : Dispatcher.CurrentDispatcher;
```

Comment:
Since you're 100% certain that tests will not use Application.Current, why not just get rid of this?